torch/nn/functional.py

```diff
@@ -273,10 +273,12 @@
         to compute the output shape. Default: ``False``
     count_include_pad: when True, will include the zero-padding in the
         averaging calculation. Default: ``True``
+    divisor_override: if specified, it will be used as divisor, otherwise
```

Comment:
I don't think this is accurate, count_include_pad is also potentially used.

Reply:
changed